site/js/i18n-resources.js

```diff
@@ -1274,6 +1284,149 @@ var resources = { // English is fallback language.
             },
         },
     }, // }}}
+
+    // Vietnamese (vi) localization {{{
+    vi: {
+        translation: {
+            "lang": {
+                "en": "Anh",
+                "fr": "Pháp",
+                "de": "Đức",
+                "ru": "Nga",
+                "pt": "Bồ Đào Nha",
+                "it": "Ý",
+                "uk": "Ukraina",
+                "nl": "Hà Lan",
+                "hu": "Hungary",
+                "es": "Tây Ban Nha",
+                "vi": "Việt",
+                "choose": "Chọn Ngôn ngữ",
+            },
+
+            "weekdays": {
+                "word next": {
+                    // FIXME: This is prepended to the day, as in English, instead of appended, as in Vietnamese.
+                    "su": "tuần sau vào",
+                    "mo": "tuần sau vào",
+                    "tu": "tuần sau vào",
+                    "we": "tuần sau vào",
+                    "th": "tuần sau vào",
+                    "fr": "tuần sau vào",
+                    "sa": "tuần sau vào",
```

Comment:
Unlike English, many languages would place the word “next” _after_ the day of the week. For example, in Vietnamese, you’d say “vào thứ 2 tuần sau” (literally, “on Monday of next week”). However, since the English order is hard-coded, I’ve translated “next” as “next week on”.

https://github.com/opening-hours/opening_hours.js/blob/eda4ac7494fd9a1d1be6b0af9830a88300d00d36/site/js/opening_hours_table.js#L40-L41

Reply:
I refactored this code to take either a format or object with formats as values, depending on the language.